scripts/rff/historical_weather_variability/county/R/county_aggregation.R

```diff
@@ -0,0 +1,248 @@
+### Aggregate PRISM to county ###
+rm(list=setdiff(ls(), c("master_path", "code_path")))
+path <- master_path
+
+coords <- read.csv(paste0(path, "replication/data/monthly_dataframes/month_ppt_coordinates.csv"))

```

Comment:
There are several string and integer constants being used extensively. Suggestion is to use variables with the constants so that it is easier to understand and follow?

Reply:
Marking resolved to submit.